### PR TITLE
fix(VM): Transparently handle computed setters

### PR DESCRIPTION
**Pull request by ahoisl**
Fixes issue #214

### CODE REVIEW

**Comment by ahoisl** [comment]
Ready to review:
- [x] unit tests included
- [x] tested manually

File: test/create-view-model.ts [hunk 1]
```diff
@@ -233,6 +246,35 @@ function tests(model) {
...
+    expect(viewModel.isPropertyDirty("prefixedTitle")).toBe(false)
+
+    viewModel.submit()
+    expect(tr).toBe(
+        "Coffee:false,interested:Putin,Madonna,Tarzan,Rocky,unobservedProp:new value,usersCount:4"
+    )
+    expect(vr).toBe(
+        "Coffee:false,interested:Putin,Madonna,Tarzan,Rocky,unobservedProp:new value,usersCount:4"
+    )
+    expect(model.title).toBe("Coffee")
```
Comment (NaridaL) [inline test/create-view-model.ts:272]: This check is redundant, it is already checked by line 266.

Comment (ahoisl) [reply test/create-view-model.ts]: True -> fixed

**Comment by NaridaL** [comment]
@ahoisl thanks!

**Comment by ahoisl** [comment]
@NaridaL anytime

**Comment by NaridaL** [comment]
I published it to npm (6.0.3). Should have been 6.0.2 ... oops 😳

**Pull request merged**